src/handbook/customer/marketing/social-media.md

```diff
@@ -0,0 +1,97 @@
+---
+navTitle: Social Media
+---
+
+# Social Media Channels
+
+Each content type has channels which are appropriate, this table sets out which content should be posted on each channel, but always mindful of the topic of the content. We should share content relevant to the each channel audience.
+
+|Content              |LinkedIn |Facebook |X        |Youtube  |Discord |Reddit |Node-RED Slack|Node-RED Discourse|
+|---                  |---      |---      |---      |---      |---     |---    |---           |---               |
+|Blog Article         |*        |*        |*        |         |        |       |              |                  |
+|Product updates      |*        |*        |*        |         |        |*      |*             |*                 |
+|Curated content      |*        |*        |*        |         |        |       |              |                  |
+|Events               |*        |*        |*        |         |*       |*      |*             |*                 |
+|Short videos         |*        |*        |*        |*        |        |       |              |                  |
+|Long format videos   |         |*        |         |*        |        |       |              |                  |
+|Downloadable content |*        |*        |*        |         |*       |       |*             |                  |
+
+## Strategy for increasing reach
+
+- Share quality content aligned with the [company strategy](/handbook/company/strategy/) and of interest to our [personas](/handbook/product/personas/#personas) and [ideal customer profile](https://docs.google.com/document/d/1krMIUJvosw8xUQog_iq_FEvI9R5WEo9ZyWUCdTb9XnQ/edit#heading=h.3rr2wuny55dl). 
+- Position ourselves as thought leaders.
+- Adapt content frequency and type to each channel.
+- Be social:
+    - Respond promptly to comments and messages.
+    - Like and share relevant content from other sources.
+- Leverage trending topics and holidays.
+
+### Scope
+
+- Channels: LinkedIn, Facebook, Twitter, Youtube.
+
+### Tactics
+
+#### Frequency
+
+- Once a work day.
+
+#### Type of content
+
+- **Own content:** 3x week. Mondays, Wednesdays, and Thursdays.
+    - 2 fresh blog articles: shared as a link to the article. The target day for review should be 3 work days before posting date, and the tile image should be requested when it's submited for review.
```

Comment:
I don't think we should commit to 2 articles in this part of the handbook, but rather state that each new piece of content will get published to social media.

That brings the question what your take is on the changelog entries on the website, are those two posted?

Reply:
Ok, I already updated the text.
Changelogs would fall into the Product update category, listed in the own content section